Extension/AbstractTableBundleExtension.php

```diff
@@ -4,8 +4,6 @@
 
 abstract class AbstractTableBundleExtension implements TableBundleExtensionInterface
```

Comment:
Looks like that the naming is a little bit mixed up in here. This thing is in the end an AbstractUqlExtension which makes its containing UqlFunctions available inside the UQL language.